.gitmodules

```diff
@@ -33,7 +33,7 @@
 	url = https://github.com/firesim/icenet.git
 [submodule "generators/block-inclusivecache-sifive"]
 	path = generators/sifive-cache
-	url = https://github.com/sifive/block-inclusivecache-sifive.git
+	url = https://github.com/ucb-bar/block-inclusivecache-sifive.git
```

Comment:
It looks like SiFive merged @colinschmidt PR in the sifive repo as well. Do we still want a fork?

Reply:
Given how responsive they were, I think not.